README.md

```diff
@@ -1837,6 +1837,10 @@ arXiv, 2020.
 - [Baldwin et al., arXiv 2021](#Baldwin21arxiv).  
 *Time-Ordered Recent Event (TORE) Volumes for Event Cameras*.
 
+- Alkendi Y., Azzam R., Ayyad A., Javed S., Seneviratne L., and Zweiri Y.
```

Comment:
There can be no empty line (space) or it will break the list. Please see the preview when committing. 
Please remove the space, so that the differences show that only one last entry was added.